### ISSUE TITLE
Compile error in postprocessing/pp-h264.c and pp-webm.c

### ISSUE THREAD

**Issue by roxlu** [meetecho/janus-gateway#622]
Enabling post processing utils results in:

`postprocessing/pp-h264.c:81:28: error: 'PIX_FMT_YUV420P' undeclared (first use in this function)
  vStream->codec->pix_fmt = PIX_FMT_YUV420P;`

Easy fixed by changing them to AV_PIX_FMT_YUV420P.

**Comment by saghul** [comment]
Hey @roxlu nice to see you here too :-)  What version of Ffmpeg / libav are you using?

**Comment by roxlu** [comment]
Hey saghul!  I'm using the latest git version of ffmpeg / libav. I got the same error about a month ago too.

**Comment by saghul** [comment]
Which of the 2? I'm not getting any problems on Debian Jessie with Ffmpeg 2.8.

**Comment by lminiero** [comment]
Yep, we also found out some defines changed, so we need some more conditions to automatically use the right ones. Will fix this shortly, hopefully tomorrow (remind me if I don't! :smile: )

**Comment by roxlu** [comment]
saghul, I'm not testing with ffmpeg

**Comment by lminiero** [comment]
@roxlu can you let me know if this patch fixes the compilation for you?

**Comment by alexamirante** [comment]
Works for me with ffmpeg latest git head
